cypress-tests/cypress/e2e/ConnectorTest/00013-SaveCardFlow.cy.js

```diff
@@ -0,0 +1,134 @@
+import createPaymentBody from "../../fixtures/create-payment-body.json";
+import confirmBody from "../../fixtures/confirm-body.json";
+import createConfirmPaymentBody from "../../fixtures/create-confirm-body.json";
+import customerCreateBody from "../../fixtures/create-customer-body.json";
+import captureBody from "../../fixtures/capture-flow-body.json";
+import getConnectorDetails from "../ConnectorUtils/utils";
+import State from "../../utils/State";
+let globalState;
+describe("Card - NoThreeDS payment flow test", () => {  
+
+    before("seed global state",  () => {
+      
+      cy.task('getGlobalState').then((state) => {
+        // visit non same-origin url https://www.cypress-dx.com
+        globalState = new State(state);
+        console.log("seeding globalState -> " + JSON.stringify(globalState));
+      })
+    })
+  
+    after("flush global state", () => {
+      console.log("flushing globalState -> "+ JSON.stringify(globalState));
+      cy.task('setGlobalState', globalState.data);
+    })
+
+  
+    context("Save card for NoThreeDS automatic capture payment- Create+Confirm", () => {
+      it("customer-create-call-test", () => {
+            cy.createCustomerCallTest(customerCreateBody, globalState); 
+      });
+  
+      it("create+confirm-payment-call-test", () => {
+        console.log("confirm -> " + globalState.get("connectorId"));
```

Comment:
why do we need to log the connector id in all confirm calls?? if not needed, please remove these logs